### PR TITLE
Transformime 2 compatibility

### PR DESCRIPTION
**Pull request by willwhitney**

### CODE REVIEW

File: src/console-text.transform.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        newLine: true
+    });
+
+    return function(mimetype, text, document) {
+        var el = doc.createElement('pre');
+
+        el.innerHTML = convert.toHtml(text);
+        return el;
+    }
+}();
```
Comment (rgbkrk) [inline src/console-text.transform.js]: Heh, same closure fun as I did in https://github.com/nteract/transformime-commonmark/commit/47956b2c0ced8320bb15624dfe5e4b79fddb5401.

File: src/index.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
-import {StreamTransformer} from './stream.transformer';
-import {TracebackTransformer} from './traceback.transformer';
-import {LaTeXTransformer} from './latex.transformer';
-import {MarkdownTransformer} from 'transformime-commonmark';
-import {PDFTransformer} from './pdf.transformer';
+import {consoleTextTransform} from './console-text.transform';
+import {LaTeXTransform} from './latex.transform';
+import {MarkdownTransform} from 'transformime-commonmark';
+import {PDFTransform} from './pdf.transform';
```
Comment (rgbkrk) [inline src/index.js]: Not all of these imports look right.

**Comment by rgbkrk** [comment]
Willing to add a simple check that uses these stitched together as part of a transformime usage? `transformime` can just be a devDependency so it can be used during testing.

File: src/index.js (hunk 2 above)
Comment (rgbkrk) [inline src/index.js]: In particular, the MarkdownTransform.

**Comment by willwhitney** [comment]
Yup, was just thinking we need real tests for these.

File: test/latex.transform.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
...xtstyle scriptstyle uncramped"><span class="mord scriptstyle uncramped"><span class="mord mathit">i</span><span class="mord mathit">n</span><span class="mord mathit" style="margin-right:0.10764em;">f</span><span class="mord mathit">t</span><span class="mord mathit" style="margin-right:0.03588em;">y</span></span></span></span><span class="baseline-fix"><span class="fontsize-ensurer reset-size5 size5"><span style="font-size:0em;">​</span></span>​</span></span></span><span class="mord mathit" style="margin-right:0.02778em;">r</span><span class="mord mathit">a</span><span class="mord mathit">c</span><span class="mord textstyle uncramped"><span class="mord">1</span></span><span class="mord textstyle uncramped"><span class="mord"><span class="mord mathit">n</span><span class="vlist"><span style="top:-0.363em;margin-right:0.05em;"><span class="fontsize-ensurer reset-size5 size5"><span style="font-size:0em;">​</span></span><span class="reset-textstyle scriptstyle uncramped"><span class="mord">2</span></span></span><span class="baseline-fix"><span class="fontsize-ensurer reset-size5 size5"><span style="font-size:0em;">​</span></span>​</span></span></span></span></span></span></span></div>';
```
Comment (rgbkrk) [inline test/latex.transform.test.js]: hahahah, yeah pretty ugly

File: test/pdf.transform.test.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        assert.equal(PDFTransform.mimetype, 'application/pdf');
+    });
+
+    it('should output the correct HTML', function() {
+        let base64PDF = "I don't have a b64'd PDF";
+        let transformed = this.t.transform(
+            {'application/pdf': base64PDF},
+            this.document);
+        return transformed.then(({el}) => {
+            assert.equal(el.outerHTML, '<a target="_blank" href="data:application/pdf;base64,I don\'t have a b64\'d PDF">View PDF</a>');
```
Comment (rgbkrk) [inline test/pdf.transform.test.js]: totes fine

File: test/latex.transform.test.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+                LaTeXTransform,
+            ]);
+    });
+
+    it('should have the text/latex mimetype', function() {
+        assert.equal(LaTeXTransform.mimetype, 'text/latex');
+    });
+
+    it('should output the correct HTML', function() {
+        let latex = '\sum\limits_{i=0}^{\infty} \frac{1}{n^2}';
```
Comment (rgbkrk) [inline test/latex.transform.test.js]: This tiny LaTeX test takes ~38ms on my machine. :confounded: 

```
  latex transform
    ✓ should have the text/latex mimetype
    ✓ should output the correct HTML (38ms)
```

I think this is a representative test though.

Comment (rgbkrk) [inline test/latex.transform.test.js]: Must have been a fluke of a single run. Not seeing the timing data anymore.

Nevermind!

**Comment by rgbkrk** [comment]
I think this is looking just fine

```
> require('.')
{ consoleTextTransform: { [Function] mimetype: 'jupyter/console-text' },
  markdownTransform: { [Function] mimetype: 'text/markdown' },
  LaTeXTransform: { [Function: LaTeXTransform] mimetype: 'text/latex' },
  PDFTransform: { [Function: PDFTransform] mimetype: 'application/pdf' } }
```

How are you feeling about it @willwhitney?

File: test/latex.transform.test.js (hunk 5 above)
Comment (willwhitney) [inline test/latex.transform.test.js]: I've had that happen. It's long.

**Comment by willwhitney** [comment]
Yup, I think this is ready to go.

**Comment by rgbkrk** [comment]
I'll let you do the honors of publishing to npm and pushing the tags to this repo.

**Comment by willwhitney** [comment]
`v0.1.0` shipped to npm.

**Pull request merged**